### ISSUE TITLE
Font awesome error

### ISSUE THREAD

**Issue by besnikh** [akulubala/vue-product-zoomer#10]
I am getting this error when I mount the component

```
vue-carousel.min.js:6 GET http://localhost:3000/assets/fonts/fontawesome-webfont.af7ae50.woff2 net::ERR_ABORTED
GET http://localhost:3000/assets/fonts/fontawesome-webfont.fee66e7.woff net::ERR_ABORTED
GET http://localhost:3000/assets/fonts/fontawesome-webfont.b06871f.ttf net::ERR_ABORTED
```

I am using different version of font-awesome.. how can exclude product-zoomer to use the outdated version or how can I get rid of this issue ? 

Thnx

**Comment by akulubala** [comment]
Hi, @besnikh , if you use webpack : 

    new CopyWebpackPlugin([
          {
            from: path.resolve(__dirname, 'node_modules/vue-product-zoomer/dist/assets'),
            to: /,
            ignore: ['.*']
          }
    ])
will copy all fonts to root path of your project.

**Comment by besnikh** [comment]
Hi @akulubala 
I do not have any experience with Webpack and I somehow try to avoid touching it since I don't quite understand how everything works. But at this time its only me dealing with this, the other guy is in vacation and we need to go live and we are happy with your plugin.

That being said, we are using nuxt, as I posted in another thread #4 I explained how to use this plugin with nuxt and avoid the error that other users are seeing while it's built for SSR. 

Now, in our project we are using latest Font-Awesome, ant this is the file in our `nuxt.config.js`
`{ rel: 'stylesheet', href: 'https://use.fontawesome.com/releases/v5.2.0/css/all.css' }`

Now, either I have to somehow change the fonts from your module and use those that we are already importing, or I have to somehow "ignore" your modules icons and find another solution. But we are heavy dependent on 5.2.0 fa fonts

Now the webpack above you mentioned, I don't believe I can include it to `nuxt.config.js ` is there any other solution to this matter ? 

I would really appreciate your help and thnx in advance,
Cheers
